src/utils/media.ts

```diff
@@ -37,3 +39,12 @@ export const getSeriesIdFromEpisode = (item: PlaylistItem) => {
 
   return null;
 };
+
+// Transfrom media items
+// Parses productId into MediaOffer[] for all cleeng offers
+export const transformMediaItem = (item: PlaylistItem) => {
```

Comment:
Recommend putting this inside `api.service.ts`

Reply:
Done